pkgs/top-level/python-packages.nix

```diff
@@ -22388,18 +22388,27 @@ EOF
   trezor_agent = buildPythonPackage rec{
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

Reply:
thanks for reviewing this! will do